bedrock/base/templates/macros.html

```diff
@@ -2,6 +2,15 @@
  # License, v. 2.0. If a copy of the MPL was not distributed with this
  # file, You can obtain one at http://mozilla.org/MPL/2.0/. -#}
 
+{% macro lazy_image(image_url, placeholder_url) -%}
+<div class="image-container">
```

Comment:
I might be overthinking it, but what if the class here was something more unique, e.g. `lazy-image-container`? Then we wouldn't have to pass a selector in at all - you'd just know that any `<img>` inside an element with class `lazy-image-container` should get observed. (This is already the case on the developer page...)